### PR TITLE
Template ScoutFS master repos for 7.3 and 7.4

### PR DESCRIPTION
**Pull request by nichenke**
Now that we have ScoutFS rpms deployed to Artifactory, we need to
configure system to pull them. These repositories are not public, so we
need to hide them behind our user/password scheme. They will use a
read-only key for our testing.

We only install these for the developer mode, as the vsm2-installer will
copy those down and manually push to nodes. We might consider on-node
repos in the future, but for now we want to keep the combinatorics under
control. The prime driver behind this is the customer installer will
copy out a release.tar.gz and then use scripts to install. We want our
developer workflows to follow this.

### CODE REVIEW

**Comment by nichenke** [comment]
Looking for both reviews before merging -- Heemstra for installer, Greg for general sanity and repo goodness.

File: templates/scoutfs-rpms.repo.j2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[scoutfs_{{ item.stage }}_{{ item.el_version }}]
+name=ScoutFS rpms from Versity {{ item.stage }}
+baseurl=https://{{ artifactory_user }}:{{ artifactory_token }}@versity.jfrog.io/versity/scoutfs-{{ item.stage }}/{{ item.el_version }}
```
Comment (gcymbalski) [inline templates/scoutfs-rpms.repo.j2]: Hrm, is the plan to have different top-level repositories for projects?

Comment (nichenke) [reply templates/scoutfs-rpms.repo.j2]: For now -- we need something non-public. I hadn't thought through if we'll have more than scoutfs RPMs to use for VSM2. Currently we have vsm-libarchive in public, then these scoutfs RPMs.

Comment (gcymbalski) [reply templates/scoutfs-rpms.repo.j2]: Sounds good- we have a 'master/weekly/daily'-rpms setup that's private, though separating things between VSM series sounds reasonable.

**Pull request merged**